lxd/db/cluster/query.go

```diff
@@ -43,9 +43,12 @@ func selectNodesVersions(tx *sql.Tx) ([][2]int, error) {
 		return []interface{}{&versions[i][0], &versions[i][1]}
 	}
 
-	stmt, err := tx.Prepare("SELECT schema, api_extensions FROM nodes WHERE pending=0")
+	stmt, err := tx.Prepare("SELECT schema, api_extensions FROM nodes WHERE state=0")
 	if err != nil {
-		return nil, err
+		stmt, err = tx.Prepare("SELECT schema, api_extensions FROM nodes WHERE pending=0")
```

Comment:
@monstermunchkin can you add a comment there explaining why this is needed (it's clearly to make cluster updates work, but it'd be good to mention it still).

Reply:
I think we have something similar in the roles logic where we need to support the older schema.